src/test-utils/package.json

```diff
@@ -4,14 +4,12 @@
   "homepage": "https://github.com/adeira/universe/tree/master/src/test-utils",
   "license": "MIT",
   "private": false,
-  "version": "0.4.0",
+  "version": "0.4.1",
   "main": "src/index",
   "sideEffects": false,
   "dependencies": {
     "@adeira/js": "^1.2.0",
-    "@babel/runtime": "^7.9.2"
-  },
-  "devDependencies": {
+    "@babel/runtime": "^7.9.2",
     "graphql": "^14.6.0"
```

Comment:
Why is this necessary? 🤔 Shouldn't it be a `peerDependency`?

Reply:
fixed